foundations/foundation-common/src/main/java/org/apache/servicecomb/foundation/common/http/HttpUtils.java

```diff
@@ -90,6 +90,9 @@ public static String uriEncodePath(String path) {
    * @return the encoded path param
    */
   public static String encodePathParam(String pathParam) {
+    if (pathParam.indexOf(';') != -1) {
+      pathParam = pathParam.substring(0, pathParam.indexOf(';'));
+    }
```

Comment:
This modification is not correct. 

Suggections:

1. upgread guava to 30.0-jre will fix this issue